wagtailmodelchoosers/widgets.py

```diff
@@ -77,33 +81,34 @@ def value_from_datadict(self, data, files, name):
     def get_endpoint(self):
         app, class_name = self.get_class_name()
 
-        from django.core.urlresolvers import reverse
-        return reverse('wagtailmodelchoosers_api_model', args=[app, class_name])
+        from django.urls import reverse
+
+        return reverse("wagtailmodelchoosers_api_model", args=[app, class_name])
 
     def get_internal_value(self, value):
-        if hasattr(value, 'pk'):
+        if hasattr(value, "pk"):
             if isinstance(value.pk, uuid.UUID):
                 return str(value.pk)
             else:
                 return value.pk
-        return ''
+        return value
```

Comment:
When `value` is None here we end up with stringified "None" as the value in form field.
```suggestion
        return value or ""
```